iocage/__init__.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/local/bin/python3.6
+# -*- coding: utf-8 -*-
+
+import re
+import sys
+import os
+
+__dirname = os.path.dirname(__file__)
+iocage_lib_dir = os.path.join(__dirname, "lib")
+sys.path = [iocage_lib_dir] + sys.path
+
+
+class Releases:
+
+   def __new__(*args, **kwargs):
+      import Releases
+      return Releases.Releases(*args[1:], **kwargs)
+
+
+class Release:
+
+   def __new__(*args, **kwargs):
+      import Releases
+      return Releases.Releases(*args[1:], **kwargs)
+
+
+class Jails:
+
+   def __new__(*args, **kwargs):
+      import Jails
+      return Jails.Jails(*args[1:], **kwargs)
+
+
+class Jail:
+   def __new__(*args, **kwargs):
+
+      import Jail
+      return Jail.Jail(*args[1:], **kwargs)
+
+
+class Host:
+
+   def __new__(*args, **kwargs):
+      import Host
```

Comment:
This file was meant to be a wrapper to access the public iocage classes on the top level of an import, e.g.

```
import libiocage
jail = Jail("foobar")
```

The common use case is that you don't require to import all to interface with the library. If you want to just start a Jail, all you need is that single class, nothing more. I was expecting the lazy imports to help keeping the startup time low.